### PR TITLE
UG: Add find command behaviour in list schedule state 

### PR DESCRIPTION
**Pull request by elaineshijie**

### CODE REVIEW

**Review by yiwen101 — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -198,7 +198,7 @@ The find command has different behaviours depending on the current list:
 1. In `STUDENTS` list:
     - Finds students whose names contain any of the given keywords.
```
Comment (pzl111) [inline docs/UserGuide.md]: Actually it finds the students whose names contains the given search string
its no longer split by multiple keywords

File: docs/UserGuide.md [hunk 2]
```diff
@@ -198,7 +198,7 @@ The find command has different behaviours depending on the current list:
...
 2. In `SCHEDULE` list:
-    - TBC
+    - Finds lessons whose names contain the given keywords.
```
Comment (pzl111) [inline docs/UserGuide.md]: similar to students, but lesson names

File: docs/UserGuide.md [hunk 3]
```diff
@@ -210,16 +210,23 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
    * Persons matching at least one keyword will be returned (i.e. `OR` search).
      e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
 2. In `SCHEDULE` list:
-    * TBC
+    * The search is case-insensitive. e.g `lesson` will match `Lesson`
+    * The order of the keywords does not matter. e.g. `Lesson Chemistry` will match `Chemistry Lesson`
```
Comment (pzl111) [inline docs/UserGuide.md]: It does matter, it checks the entire search string to see if lesson name contains the search string

File: docs/UserGuide.md [hunk 4]
```diff
@@ -210,16 +210,23 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
...
+    * Only the name is searched.
+    * Only full words will be matched e.g. `Physics` will match `Phy`
```
Comment (pzl111) [inline docs/UserGuide.md]: this isnt valid since we find by search string

File: docs/UserGuide.md [hunk 5]
```diff
@@ -210,16 +210,23 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
...
+    * Lessons matching at least one keyword will be returned (i.e. `OR` search).
```
Comment (pzl111) [inline docs/UserGuide.md]: irrelevant already

**Review by pzl111 — CHANGES_REQUESTED** [review]
Minor change to description

File: docs/UserGuide.md (hunk 1 above)
Comment (elaineshijie) [reply docs/UserGuide.md]: Thank you, made changes in new commit.

File: docs/UserGuide.md (hunk 2 above)
Comment (elaineshijie) [reply docs/UserGuide.md]: Thank you, made changes in new commit

File: docs/UserGuide.md (hunk 3 above)
Comment (elaineshijie) [reply docs/UserGuide.md]: Thank you, made changes in new commit

File: docs/UserGuide.md (hunk 4 above)
Comment (elaineshijie) [reply docs/UserGuide.md]: Thank you, made changes in new commit

File: docs/UserGuide.md (hunk 5 above)
Comment (elaineshijie) [reply docs/UserGuide.md]: Thank you, made changes in new commit

File: docs/UserGuide.md [hunk 6]
```diff
@@ -196,30 +196,37 @@ Examples:
...
+   * Keyword specified has to be the first part of the name
+     e.g. `Hans` will return `Hanso Gruber` but not `Lee Hans`
 2. In `SCHEDULE` list:
-    * TBC
+    * The search is case-insensitive. e.g `lesson` will match `Lesson`
+    * Only the name is searched.
+    * Lessons matching part of the keyword will be returned (i.e. `OR` search).
+      e.g. `Lesson Chem` will return `Lesson Chemistry`, `Lesson Chem at Bishan`
+    * Keyword specified has to be the first part of the name
+        e.g. `Blk 12` will return `Blk 123 Bedok Chemistry` but not `Bishan Blk 12`
```
Comment (pzl111) [inline docs/UserGuide.md]: both actually returns
because it simply searches if the lesson name contains the search string

**Review by pzl111 — CHANGES_REQUESTED** [review]
minor issue

File: docs/UserGuide.md (hunk 6 above)
Comment (elaineshijie) [reply docs/UserGuide.md]: okies made the necessary changes in the most recent commit

**Review by pzl111 — APPROVED** [review]
LGTM

**Pull request merged**